unblob/handlers/compression/lzh.py

```diff
@@ -0,0 +1,106 @@
+import io
+from typing import List, Optional
+
+from structlog import get_logger
+
+from ...file_utils import Endian
+from ...models import StructHandler, ValidChunk
+
+logger = get_logger()
+
+PADDING_LEN = 2
+LEVEL_IDENTIFIERS = [0, 1, 2]
+METHOD_IDS = [
+    b"-lh0-",
+    b"-lzs-",
+    b"-lz4-",
+    b"-lh1-",
+    b"-lh2-",
+    b"-lh3-",
+    b"-lh4-",
+    b"-lh5-",
+    b"-lh6-",
+    b"-lh7-",
+    b"-lh8-",
+    b"-lhd-",
+]
+
+
+class LZHHandler(StructHandler):
```

Comment:
Now it looks cool! I just have 1 missing piece I try to understand:

We have nice test files for all levels (0-1-2). 
I think they look completely the same (each level is one `.lzh` file containing three text files). HOWEVER, the output is different. 
* In level 0-1 case, the `LZHHandler` is running 3 different times, extracting the each text file separately 
* In level 2, it runs only once, extracting the 3 files in 1 step. This is sooo much cleaner.

Why? :sweat_smile:

Reply:
Hands down best reviewer here. You are totally right and this was due to a wrong calculation of header size when in level 2 (I was converting as big endian instead of little endian). It matched the chunk size out of pure luck. It's fixed now and we have a separate chunk for each LZH stream.